src/com/facebook/buck/ide/intellij/IjProjectCommandHelper.java

```diff
@@ -190,7 +206,10 @@ public ExitCode parseTargetsAndRunProjectGenerator(List<String> arguments)
 
       return ExitCode.SUCCESS;
     }
-
+    ExitCode result = initGoWorkspace(targetGraphAndTargets);
```

Comment:
So it will affect all the invocations of `buck project` even if it's not related to Go?

Reply:
No, `initGoWorkspace` only scans go_library rules. If there is no such rule in the workspace, it is a no-op.

Reply:
From the way it looks it should implemented as a separate command.

For which IDE this command is implemented? Is that just copying files?

Reply:
This has been tested for GoLand and Visual Studio Code. Yes, it is just copying files. This is sufficient for these IDEs to work. Go is more like "convention over configuration" style. So it has strong conventions about where packages should be put, rather than relying on configuration files to tell the IDEs and Go toolchain where to search for packages.